spec/facility_spec.rb

```diff
@@ -2,25 +2,171 @@
+      expect(@facility_1.address).to eq('2855 Tremont Place Suite 118 Denver CO 80205')
+      expect(@facility_1.phone).to eq('(720) 865-4600')
+      expect(@facility_1.services).to eq([])
     end
   end
 
   describe '#add service' do
     it 'can add available services' do
-      expect(@facility.services).to eq([])
-      @facility.add_service('New Drivers License')
-      @facility.add_service('Renew Drivers License')
-      @facility.add_service('Vehicle Registration')
-      expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+      expect(@facility_1.services).to eq([])
+      @facility_1.add_service('New Drivers License')
+      @facility_1.add_service('Renew Drivers License')
+      @facility_1.add_service('Vehicle Registration')
+      expect(@facility_1.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+    end
+  end
+
+  describe '#registration_date' do
+    it 'date registered' do
+      expect(@cruz.registration_date). to be(nil)
+    end
+  end
+
+  describe '#registered_vehicles' do
+    it 'is a list of registered vehicles' do
+      expect(@facility_1.registered_vehicles).to eq([])
+    end
+  end
+
+  describe '#collected_fees' do
+    it 'fees collected by dmv facility for registration of vehicle' do
+      expect(@facility_1.collected_fees).to eq(0)
+    end
+  end
+
+  describe '#register_vehicle' do
+    it 'adds vehicle to list of registered_vehicles'do
+      expect(@facility_1.register_vehicle(@cruz)).to eq([@cruz])
+    end
+  end
+
+  describe '#new_registration_date' do
+    it 'logs the time of vehicle registered' do
+      # require 'pry'; binding.pry
+      expect(@cruz.new_registration_date).to eq(@cruz.new_registration_date)#check this out soon
+    end
+  end
+
+  describe '#plate_type' do
+    it 'returns plate_type category' do
+      # require 'pry'; binding.pry
+      expect(@cruz.plate_type).to eq(:regular)
+    end
+  end
+
+  describe '#collected_fees'do
+    it 'collects car registration fees' do
+      expect(@facility_1.register_vehicle(@cruz)).to eq([@cruz])
+      expect(@facility_1.registered_vehicles).to eq([@cruz])
+      expect(@facility_1.rcollected_fees(@cruz)).to eq(100)
+    
+      expect(@facility_1.register_vehicle(@camaro)).to eq([@cruz, @camaro])
+      expect(@camaro.new_registration_date).to eq(@camaro.new_registration_date)#check this out soon
+      expect(@facility_1.registered_vehicles).to eq([@cruz, @camaro])
+      expect(@facility_1.rcollected_fees(@camaro)).to eq(125) 
+      expect(@camaro.plate_type).to eq(:antique)
+
+      expect(@facility_1.register_vehicle(@bolt)).to eq([@cruz, @camaro, @bolt])
+      expect(@bolt.new_registration_date).to eq(@bolt.new_registration_date)#check this out soon
+      expect(@bolt.plate_type).to eq(:ev)
+      expect(@facility_1.registered_vehicles).to eq([@cruz, @camaro, @bolt])
+      expect(@facility_1.rcollected_fees(@bolt)).to eq(325) 
```

Comment:
Nice work asserting the values of `registered_vehicles` and `collected_fees` every time you registered another vehicle 👍🏻